app/controllers/Checkout.scala

```diff
@@ -67,20 +67,16 @@ class Checkout(fBackendFactory: TouchpointBackends, commonActions: CommonActions
 
       val matchingPlanList: Option[PlanList[ContentSubscription]] = {
 
-        val testOnlyPlans = if (tpBackend == fBackendFactory.Test) List(catalog.weekly.zoneB.plansWithAssociations) else List.empty
```

Comment:
I think this has been hanging around since the GW migration - I don't think anyone is testing with Guardian Weekly Zone B plans anymore.